packages/shopping-cart/src/shopping-cart-endpoint.ts

```diff
@@ -145,6 +145,10 @@ export interface ResponseCartProduct {
 	extra: ResponseCartProductExtra;
 	uuid: string;
 	cost: number;
+	cost_before_coupon?: number;
```

Comment:
You marked all these as optional, but I'm pretty sure they're all always set. Is there a reason you marked them optional?

Reply:
@sirbrillig Just because I was not sure they're always set. If you're sure, I would not mind marking them non-optional.

Reply:
👍  It's not a big deal. I was just curious. I'll verify and if I'm right I'll make a PR. Thanks!